docs/ECLLanguageReference/ECLR_mods/Value-Data.xml

```diff
@@ -24,6 +24,9 @@
   role="bold">Working with BLOBs</emphasis> for more information on this
   subject.</para>
 
+  <para>The upper size limit for any DATA value is 4GB (2^32), although you
```

Comment:
I would remove the comment about running out of memory.  Same goes for all the other items.